packages/manager/apps/pci-public-ip/src/components/list/NetworkSecurityAction.tsx

```diff
@@ -11,14 +11,31 @@ import {
 import { OsdsButton, OsdsText } from '@ovhcloud/ods-components/react';
 import { useTranslation } from 'react-i18next';
 import { useNavigation } from '@ovh-ux/manager-react-shell-client';
+import { IPsTabName } from '@/constants';
 
-export const NetworkSecurityAction = ({ projectId }: { projectId: string }) => {
+type NetworkSecurityActionProps = {
+  projectId: string;
+  ipTabName?: IPsTabName;
+};
+
+export const NetworkSecurityAction = ({
+  projectId,
+  ipTabName = IPsTabName.FLOATING_IP_TAB_NAME,
```

Comment:
nitpick :

to make this component cleaner instead of using an ipTabName props you can use a boolean isFloatingIp

this way the component is not tied to the ux concept of tabs but is tied to the business logic of "is the ip a floating ip"

Reply:
Convinced, fix done :)